keep/providers/jira_provider/jira_provider.py

```diff
@@ -328,6 +383,19 @@ def _extract_project_key_from_board_name(self, board_name: str):
             )
         else:
             raise Exception("Could not fetch boards: " + boards_response.text)
+        
+    def _extract_issue_key_from_ticket_id(self, ticket_id: str):
+        issue_key = requests.get(
+            f"{self.jira_host}/rest/api/2/issue/{ticket_id}",
+            auth=self.__get_auth(),
+            headers={"Accept": "application/json"},
+            verify=False,
+        )
+
+        if issue_key.status_code == 200:
+            return issue_key.json()["key"]
+        else:
+            raise Exception("Could not fetch issue key: " + issue_key.text)
```

Comment:
What's the difference between ticket_id and issue_key? why don't we just keep the issue_key instead of extracting it?

Reply:
Changed to ticket_id to issue_id